### PR TITLE
Fix phpunit version used by travis & fix test bootstrap

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
This should at least remove the need for workarounds in the tarvis configuration. The tests in its current invokation are still broken, however, when executing phpunit in vendor/squizlabs/php_codesniffer, everytinh is fine. I guess this is due some autoloading magic. In my CS projects I just invoke phpunit in that directory, but don't have a central tool like ant...

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea
```
Comment (avdg) [inline .gitignore]: This might be placed better in `.git/info/exclude` if it's done for personal stuff

Comment (avdg) [reply .gitignore]: See https://help.github.com/articles/ignoring-files/

Comment (mmoll) [reply .gitignore]: OK

**Comment by avdg** [comment]
Build is failing mainly due to the removal of the autoloader from phpcs itself. Really need an investigation on how autoloaders works (or should work) so there is no issue with all projects depending on it (as the phpcs autoloader got removed due to issues with an other project).

**Comment by mmoll** [comment]
This is kind of a kludge, but "works for me"[tm]: https://github.com/mmoll/mo4-coding-standard/blob/b21e18a824c7967ce118032fb9abbac29726036b/.travis.yml

File: composer.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     },
     "minimum-stability": "dev",
     "require": {
-        "squizlabs/php_codesniffer": "3.*"
+        "squizlabs/php_codesniffer": "3.*",
+        "phpunit/phpunit": "^4.0 || ^5.0 || ^6.0"
```
Comment (xalopp) [inline composer.json]: I recommend to `require` phpunit as `require-dev`

Comment (mmoll) [reply composer.json]: done.

**Comment by xalopp** [comment]
@avg @djoos 
could you review and hopefully approve & merge this PR, so the green light of Travis CI shines again on this project? 😃

**Comment by avdg** [comment]
I cannot give an official rubber duck stemple sample but I can see the build builds nicely and the patch is quite clean and full of nice candy, and I guess that's all we should care about for now (unless someone can spot some dirty spots in the scenery of this patch).

**Comment by djoos** [comment]
Agreed - let’s get the tests here green again and we’ll take it from there!

**Comment by djoos** [comment]
Thanks for your contribution @mmoll!

**Pull request merged**